[rospy.client][INFO] 2020-02-08 18:29:50,662: init_node, name[/ros_test], pid[29332]
[xmlrpc][INFO] 2020-02-08 18:29:50,662: XML-RPC server binding to 0.0.0.0:0
[rospy.init][INFO] 2020-02-08 18:29:50,662: ROS Slave URI: [http://huai:38999/]
[xmlrpc][INFO] 2020-02-08 18:29:50,662: Started XML-RPC server [http://huai:38999/]
[rospy.impl.masterslave][INFO] 2020-02-08 18:29:50,663: _ready: http://huai:38999/
[xmlrpc][INFO] 2020-02-08 18:29:50,667: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2020-02-08 18:29:50,668: Registering with master node http://localhost:11311
[rospy.init][INFO] 2020-02-08 18:29:50,763: registered with master
[rospy.rosout][INFO] 2020-02-08 18:29:50,763: initializing /rosout core topic
[rospy.rosout][INFO] 2020-02-08 18:29:50,765: connected to core topic /rosout
[rospy.simtime][INFO] 2020-02-08 18:29:50,765: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-02-08 18:29:51,055: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-02-08 18:29:53,157: topic[/Lines2d] adding connection to [/tracking_node], count 0
[rospy.internal][INFO] 2020-02-08 18:30:01,699: topic[/camera/infra1/image_rect_raw] adding connection to [http://huai:40987/], count 0
[rospy.internal][INFO] 2020-02-08 18:45:03,615: topic[/rosout] removing connection to /rosout
[rospy.core][INFO] 2020-02-08 18:45:03,761: signal_shutdown [signal-2]
[rospy.registration][WARNING] 2020-02-08 18:45:03,761: cannot unregister with master due to network issues
[rospy.internal][INFO] 2020-02-08 18:45:03,761: topic[/Lines2d] removing connection to /tracking_node
[rospy.internal][INFO] 2020-02-08 18:45:03,761: topic[/camera/infra1/image_rect_raw] removing connection to http://huai:40987/
[rospy.internal][WARNING] 2020-02-08 18:45:03,764: Unknown error initiating TCP/IP socket to huai:51639 (http://huai:40987/): Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.read_header()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 653, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2020-02-08 18:45:03,764: Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 841, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.impl.masterslave][INFO] 2020-02-08 18:45:03,764: signal-2
[rospy.core][INFO] 2020-02-08 18:45:03,880: signal_shutdown [atexit]
